app/components/forms/wizard/basic-details-step.js

```diff
@@ -142,7 +142,7 @@ export default Component.extend(FormMixin, EventWizardMixin, {
       return parseInt($('.ui.form').form('get value', 'ticket_min_order'), 10) <= parseInt($('.ui.form').form('get value', 'ticket_max_order'), 10);
     };
     $.fn.form.settings.rules.checkValidTimeDifference = () => {
-      return moment($('[name=start_time]')[0].value, 'HH:mm').isBefore(moment($('[name=end_time]')[0].value, 'HH:mm'));
+      return $('[name=start_date]')[0].value === $('[name=end_date]')[0].value ? moment($('[name=start_time]')[0].value, 'HH:mm').isBefore(moment($('[name=end_time]')[0].value, 'HH:mm')) : true;
```

Comment:
Use &&

Reply:
@iamareebjamal sir please explail this once , i am unable to decode what you are saying.

Reply:
Join the 7 PM meeting. What is your email?

Reply:
@iamareebjamal yasharyan1099@gmail.com,